README.md

```diff
@@ -20,12 +23,16 @@ $ conda activate base
 $ conda install \
     --override-channels \
     -c conda-canary/label/conda-conda-pr-11193 \
-    -c conda-canary/label/conda-libmamba-solver-pr-3 \
+    -c conda-canary/label/conda-libmamba-solver-dev \
```

Comment:
Where does the dev come from? Don't see it here: https://anaconda.org/conda-canary/conda-libmamba-solver

Reply:
Technically from [here](https://github.com/conda-incubator/conda-libmamba-solver/blob/eb8c0694e6e3846d14bd3a80207ff858376dc3da/.github/workflows/builds-canary.yaml#L36)

Reply:
Ah, d'oh, makes sense!